packages/flutter/test/material/floating_action_button_theme_test.dart

```diff
@@ -22,17 +22,25 @@ void main() {
       ),
     ));
 
+    final ElevatedButton elevatedButton = tester.widget(find.byType(ElevatedButton));
+    final BoxConstraints buttonConstraints = BoxConstraints(
+      minWidth: elevatedButton.style!.minimumSize!.resolve(enabled)!.width,
+      minHeight: elevatedButton.style!.minimumSize!.resolve(enabled)!.height,
+      maxWidth: elevatedButton.style!.maximumSize!.resolve(enabled)!.width,
+      maxHeight: elevatedButton.style!.maximumSize!.resolve(enabled)!.height,
```

Comment:
Here and elsewhere, can you store some of these, like `elevatedButton.style`,  in local variables for readability? You would not have to have so many `!`s either. :)